tuf/api/metadata.py

```diff
@@ -786,13 +754,11 @@ def verify_length_and_hashes(self, data: Union[bytes, BinaryIO]):
 class Timestamp(Signed):
     """A container for the signed part of timestamp metadata.
 
-    Attributes:
-        meta: A dictionary that contains information about snapshot metadata::
-
-            {
-                'snapshot.json': <MetaFile INSTANCE>
-            }
+    Timestamp contains information about the snapshot Metadata file.
 
+    Attributes:
+        meta: A dictionary of filenames to MetaFiles. Only valid key value is
+            the snapshot filename, as defined by specification.
```

Comment:
```suggestion
        meta: A dictionary of filenames to MetaFiles. The only valid key value
            is the snapshot filename, as defined by the specification.
```
IMNSHO these additional words increase ease of reading without increasing line count, but feel free to ignore.